src/FFmpegReader.cpp

```diff
@@ -569,13 +573,13 @@ void FFmpegReader::Open() {
 			CheckFPS();
 		}
 
+		// Mark as "open"
+		is_open = true;
+
 		// Seek back to beginning of file (if not already seeking)
 		if (!is_seeking) {
 			Seek(1);
```

Comment:
Since the Seek() method can actually Close() and re Open() this reader, it's important we mark the reader as `is_open = True` before doing this... so our Close method can actually clean up the open reader.